CHANGELOG.md

```diff
@@ -83,6 +83,11 @@ file an issue, so that we can see how to handle this.
 * Consider calling `fairroot_check_root_cxxstd_compatibility()`
   in your `CMakeLists.txt`.
 * `fairsoft-config` isn't searched for and not needed any more.
+* `fEvtHeader` member variable now is a private unique pointer owned by `FairRun`. To access
+  the event header, please use the public member function `GetEventHeader()`.
+* `FairRun::SetEventHeader(FairEventHeader*)`, 
+use `FairRun::SetEventHeader(std::unique_ptr<FairEventHeader> EvHeader)`, which 
+indicates the ownership transferring.
```

Comment:
1. Some really small suggestions:

```suggestion
* `fEvtHeader` member variable now is a private unique pointer owned by `FairRun`. To access
  the event header, please use the public member function `GetEventHeader()`.
* `FairRun::SetEventHeader(FairEventHeader*)` got deprecated, 
  use `FairRun::SetEventHeader(std::unique_ptr<FairEventHeader> EvHeader)`, which 
  indicates the ownership transferring.
```

2. Please reoder a bit:
   * The change of fEvtHeader is actually breaking (because derived classes can't any longer acces it). Could you move that bullet point under the "Breaking Changes" subsection?
   * And the other one is a deprecation. We also have a subsection for deprecationbs. Cizld you move it there?

Reply:
Sure, no problem.

Reply:
Fixed.